src/i18n/zh.js

```diff
@@ -191,6 +191,10 @@ const zh = {
             format: "格式",
             formatted_body: "格式化的数据",
             algorithm: "算法",
+            url: "URL",
```

Comment:
Don't add any translation if it's not really locallized. Synapse-admin will always fall back to english.

Reply:
I was expecting it to be, but that wasn't what I observed, it defaults to auto-generated value instead of the one customized in localization, e.g. "Event json content info mimetype" and "Event json content url" instead of MIME/URL specified in English localization.

Reply:
Removed extra localizations, but you have to fix precedence issue I described above in separate patch.

Reply:
Ah, you're right. Nevertheless I will fix this first. By leaving out non-translated items from the translations files it's easier to spot what still needs to be properly translated.